### PR TITLE
Adds rotating file log

### PR DESCRIPTION
**Pull request by ntanck**

### CODE REVIEW

**Comment by ntanck** [comment]
Ran for a few days on a couple of BBBs, showed no signs of failure or uncaught exceptions. No working parts were altered.

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[flake8]
+max-line-length = 119
+max-complexity = 18
+exclude =
+    .git,
+    __pycache__
+ignore=W503,E203,F405,E722,F403,E402
```
Comment (pnallin) [inline .flake8:7]: .gitignore ?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 119
+target-version = ['py37']
+include = '\.pyi?$'
```
Comment (pnallin) [inline pyproject.toml:4]: .gitignore ?

**Review by ntanck — COMMENTED** [review]
Checked all imports

**Review by pnallin — APPROVED** [review]

**Pull request merged**